ComponentKit/Core/CKComponentMemoizer.mm

```diff
@@ -29,6 +29,38 @@ size_t operator ()(CKMemoizationKey a) const {
   };
 }
 
+struct CKLayoutMemoizationKey {
+  CKComponent *component;
+  CKSizeRange thatFits;
+  CGSize parentSize;
+
+  struct Hash {
+    size_t operator ()(CKLayoutMemoizationKey a) const {
+      NSUInteger subhashes[] = {
+        a.component.hash,
+        std::hash<CGFloat>()(a.thatFits.min.width),
```

Comment:
That would be great once https://github.com/facebook/componentkit/pull/285 lands